lib/gprocess.c

```diff
@@ -384,6 +393,12 @@ g_process_check_cap_syslog(void)
     }
 }
 
+gboolean
+g_process_is_cap_enabled(void)
+{
+  return process_opts.enable_caps;
+}
```

Comment:
this seems like a bit redundant to me, I mean we have a boolean parameter in addition caps being NULL or not NULL. I could accept the redundancy, however as it seems the --no-caps argument manipulates caps and not this variable, thus we'd still try to use caps.

I am a bit tired now and may be missing something though.

Reply:
The `--no-caps` manipulates `enable_caps` to `FALSE`.  The `caps` & `!caps` won't fly anymore, because actually we need 3 state:
* default caps
* caps from user
* no caps

Before this patch, this 3 state was solved by initializing with `default caps` and when user overwrite it everything was as expected.
But because I need to have the `debug_flag` before setting its default value, that trick won't help anymore.

Reply:
@Kokan : add these information to the commit message (where `enable_caps` is introduced)

Reply:
Sure.

Reply:
@Kokan : thanks!!